src/style.css

```
@@ -0,0 +1,21 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 12px;
+  width: 13px;
+  margin: 2px 1.5px;
+  margin-right: 5.5px;
```

```suggestion
  height: 16px;
  width: 16px;
  margin-right: 4px;
  background-position: center;
```
redundant margins around stars.
better to set proper height and width, background position, and make margin from one side

also reset margin-right for last star in row